developers/developers.md

```diff
@@ -107,6 +107,14 @@ The behavior of each motor will be defined by its assigned contributions to each
 
 <img src="/images/bluerov-frame.png" class="img-responsive img-center" style="max-height:250px;">
 
+## Troubleshooting
+
+1. If something goes wrong while compiling:
+    1. Check the **Compiling** section to set up your environment.
+    2. ArduSub is only compatible with GCC 4.9 for non-Linux systems. Be sure that "gcc-arm-none-eabi 4.9" is your "PATH" env with: `echo $PATH | grep gcc-arm-none-eabi`.
+        1. It's also possible to add custom paths for the compiler with `export PATH=/your_path/gcc-arm-none-eabi-4_9-2015q3/bin/:$PATH`, and make sure to run "WAF" or "MAKE" from the first step.
+2. To report any problem, you can use or [github issue page](https://github.com/ArduPilot/ardupilot/issues).
```

Comment:
This link is redundant with the footer, and I prefer the issue go to the docs. It means that we know people are using these instructions and have a problem with it, and we know that the problem is with these instructions.